blogposts/2021/dev-tool-time-liz-fong-jones.md

```diff
@@ -29,25 +29,25 @@ Liz's desk isn't the only place she works, though. Despite an ergonomic setup, L
 
 In her terminal, which runs [Bash](https://www.gnu.org/software/bash/), she uses [Powerline](https://powerline.readthedocs.io/en/latest/index.html) with the default config to manage complexity and add nice-to-have features, such as visiblity on current host and background jobs. Using this set of tools makes Liz's setup easily accessible in new computing environments.
 
 ## Observability
 
-Liz's dev workflow follows a structure that might be familiar for many devs. She uses [Git](https://git-scm.com/) for version control, works on a local copy of a repo in her editor (GNU nano), and pushes to a remote that's hooked up with CircleCI to run a variety of actions. Differently from some devs, she doesn't rely on her local editor to run formatting steps or lint code. Instead, [CircleCI](https://circleci.com/) takes care of these steps, along with deployment and other actions.
+Liz's dev workflow follows a structure that might be familiar for many devs. She uses [Git](https://git-scm.com/) for version control, works on a local copy of a repo in her editor (GNU nano), and pushes to a remote that's hooked up with [CircleCI](https://circleci.com/) to run a variety of actions. Differently from some devs, she doesn't rely on her local editor to run formatting steps or lint code. Instead, CircleCI] takes care of these steps, along with deployment and other actions.
 
-This automation goes hand-in-hand with Liz's attention to [observability](https://www.honeycomb.io/what-is-observability/), which emphasizes the capacity to introspect a system and understand its internal state. Automated builds and infrastructure configuration, using tools such as [Chef](https://www.chef.io/) and [Terraform](https://www.terraform.io/), increase the transparency of the systems with which Liz regularly works. As part of her work for [Honeycomb](https://www.honeycomb.io/), Liz uses Honeycomb, and shifting into its interface to understand workflows, processes, and debug errors or failures is a key part of her workflow.
+This automation goes hand in hand with Liz's attention to [observability](https://www.honeycomb.io/what-is-observability/), which emphasizes the capacity to introspect a system and understand its internal state. Automated builds and infrastructure configuration, using tools such as [Chef](https://www.chef.io/) and [Terraform](https://www.terraform.io/), increase the transparency of the systems with which Liz regularly works. As part of her work for [Honeycomb](https://www.honeycomb.io/), Liz uses Honeycomb, and shifting into its interface to understand workflows, processes, and debug errors or failures is a key part of her workflow.
```

Comment:
I usually go by the rule of using a hyphen where the phrase appears before a noun (acting as an adjective). I can get this into our handbook as I see it isn't specified: https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#dashes-and-hyphens)